src/main/resources/META-INF/plexus/components.xml

```diff
@@ -17,6 +22,7 @@
        <package>org.apache.maven.plugins:maven-jar-plugin:2.6:jar,${project.groupId}:${project.artifactId}:${project.version}:build</package>
        <pre-integration-test>${project.groupId}:${project.artifactId}:${project.version}:start</pre-integration-test>
        <integration-test>org.apache.maven.plugins:maven-failsafe-plugin:2.19.1:integration-test</integration-test>
+       <!-- post-integration-test is opportunity for code coverage, such as jacoco, to collect statistics from running container -->
        <post-integration-test></post-integration-test>
        <verify>${project.groupId}:${project.artifactId}:${project.version}:stop,org.apache.maven.plugins:maven-failsafe-plugin:2.19.1:verify</verify>
```

Comment:
> I was guided in my choices by what other container technologies use; Cargo recommends associating start goal with pre-integration-test and stop goal with post-integration-test. Tomcat maven plugin recommends associating run-war-only goal with pre-integration-test and shutdown goal with post-integration-test.

That is exactly now what I suggest, too.`docker:start` in *pre-integration-test* and `docker:stop` in *post-integration-test*. My problem to moving `docker:stop` to *verify* is that you can easily call `mvn integration-test` which won't run *verify* and hence doesnt stop the container.

I know there wont be a perfect solution for the lifecycle choice, but as an alternative you could use a [preStop](http://fabric8io.github.io/docker-maven-plugin/docker-start.html#wait-during-startup-and-shutdown) command to get data out of the container before it stops. Or you mount the the directory where jacoco dumps its data as a volume dir within `target`. 

wdyt, would this be ok ?

Btw, I introduced two other packagings, too: 'docker-build' for only building images and 'docker-tar' for creating a docker tar as artifact (Dockerfile + support files).

Thanks btw for the nice PR, makes things quite simpler :)